src/main/java/lottoauto/model/WinningLotto.java

```diff
@@ -1,44 +1,58 @@
 package lottoauto.model;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
 public class WinningLotto {
 
-    private final Lottos lottos;
-
     private final List<Integer> winningLotto;
+    private final int bonusNumber;
 
-    public WinningLotto(Lottos lottos, List<Integer> winningLotto) {
-        winningNumberValidation(winningLotto);
-        this.lottos = lottos;
+    public WinningLotto(List<Integer> winningLotto, int bonusNumber) {
+        winningNumberValidation(winningLotto, bonusNumber);
+        winningLotto.add(bonusNumber);
         this.winningLotto = winningLotto;
+        this.bonusNumber = bonusNumber;
     }
 
-    public static WinningLotto of(Lottos lottos, List<Integer> winningNumbers) {
-        return new WinningLotto(lottos, winningNumbers);
+    public WinningReward compare(Lotto lotto) {
+
+        boolean existBonus = lotto.getNumbers().contains(bonusNumber);
```

Comment:
```suggestion
        boolean existBonus = lotto.contains(bonusNumber);
```

디미터 법칙을 지키지 않고 있는 것 같아요
lotto가 직접 처리할 수 있을 것 같아요

Reply:
아..자꾸 잊게 되네요..감사합니다!